### PR TITLE
Provide a helpful error message when passing a string to label

### PR DESCRIPTION
**Pull request by drapergeek**

### CODE REVIEW

**Comment by drapergeek** [comment]
>I think this LGTM. Another possibility would be to make that function translate it into a keyword list as a shortcut since I imagine changing the label text is a fairly common occurrence


I've debated this and I'd love your opinion (and @dbernheisel) on the implications of that. My reasoning for not allowing that was simply to make it an easier "one option" interface. I'm primarily trying to keep it simple for designers so they can quickly follow it without needing to know the library in depth.

**Comment by paulcsmith** [comment]
I think converting it to label text falls right in the sweet spot of useful magic because it's hard to use wrong, and easy to understand at first glance.

For example, I wouldn't do something like: `label: "my-css-class"` because that just looks weird (to me at least). I think most designers would agree. So IMO I think it would be hard to misuses and it seems easy to understand when reading.

**Comment by dbernheisel** [comment]
If the docs are clear, I could go either way, but I think the extra magic makes sense.

**Comment by drapergeek** [comment]
Seems reasonable, updated!

File: test/formulator_test.exs [hunk 1]
```diff
@@ -31,6 +31,13 @@ defmodule FormulatorTest do
 
       assert label |> to_string == ~s(<label class="control-label" for="_name">Name</label>)
     end
+
+    test "passing the label: 'text' option allows for overring the label" do
```
Comment (paulcsmith) [inline test/formulator_test.exs]: overring -> overriding :)

**Pull request merged**